docs/UserGuide.md

```diff
@@ -1,134 +1,162 @@
-
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
-
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
-
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
-
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
-
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
-
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+#### Adding a task: `add`
+To add a new task to Taskell, use the add command.<br>
+Formats: 
+- `add TASK ITEM` <br>
+This format can be used to add floating tasks without any deadlines.<br>
+> Example: `add read Harry Potter Book` <br>
+- `add TASK ITEM by [DATE]` <br>
+This format can be used to add events held on a certain day.<br>
+> Example: `add buy MA1101R textbook by today` <br>
+> Example: `add do CS2100 assignment by 10th August`<br>
+> Example: `add go shopping by tomorrow`<br>
+- `add TASK ITEM by [DATE][TIME]` <br>
+This format can be used to add tasks with a stipulated deadline.<br>
+> Example: `add do lab homework on Friday 7pm` <br>
+-  `add TASK ITEM on [DATE]` <br>
+The use of word �on� gives more flexibility for you.<br>
+> Example: `add schedule meeting on Thursday` <br>
+- `add TASK ITEM on [DATE][TIME]` <br>
+> Example: `add meet teacher on Friday 7pm` <br>
+
+> Take Note! Dates need to refer to the current week dates the task was keyed in. Eg. 
+> "tomorrow" and "thursday" is valid but "next Saturday" is invalid. Only the words "on"
```

Comment:
this is a big bit of information, so, may be you can add this line right under the `add` description to get user attention rather than use it in a quoted text.